keyvault/client/src/main/java/com/azure/keyvault/models/SecretBackup.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault.models;
+
+import com.azure.common.implementation.Base64Url;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SecretBackup {
```

Comment:
I don't want this to be a public class the Backup / Restore methods on SecretClient should just return / take byte[]

Reply:
Made package-private.